data/map.txt

```diff
@@ -29124,6 +29246,35 @@ planet "New Switzerland"
 		threshold 2000
 		fleet "Small Republic" 11
 
+planet "New Tortuga"
+	attributes pirate "core pirate" core religious factory
+	landscape land/fog6
+	description `A planet of violent storms and frequent earthquakes, New Tortuga would turn away anyone unless they have a reason to hide. The few settlements on the planet are subterranean cave cities far from any fault lines. Thick blast doors block all entrances, and vast pumping systems remove water after the frequent supertyphoon-strength tropical storms.`
+	description `Most of the inhabitants of this planet are Syndicate dissidents who support themselves through manufacturing. The Syndicate often destroys pirate factories at their borders, but their fleets cannot penetrate as far as New Tortuga. Here, shipyards, factories, and outfitters provide an uninterrupted supply.`
+	spaceport `As you walk through the spaceport, you notice a surprising number of religious buildings scattered between the pirate bars and black market stores. Most of the shops are run not by pirates, but by ordinary folk, frequently wearing religious garb. They interact with pirates as customers and guests rather than as an oppressive government.`
```

Comment:
I liked this section a lot. Having the pirates here be very religious is a nice cultural touch that provides a bit of humanity to a faction we typically see as mostly evil.

Alternative Spaceport (1) Description 1: "As you walk through the spaceport, you notice a surprising number of religious buildings scattered between the pirate bars and black-market stores. Most of the shops are run not by pirates, but by ordinary folk, many wearing religious garb. The citizens here interact with pirates with friendliness and warmth rather than the hostility and fear you normally see."

Alternative Spaceport (1) Description 2: "As you walk through the spaceport, amidst the spacebars and black-markets, you notice a surprising variety of religious buildings. There are a variety of churches, synagogues, and mosques, as well as a smattering of religious buildings you can’t quite remember the names of. Most of the black-market shops are run not by pirates, but by ordinary folk, many wearing religious garb. The citizens here interact with pirates with friendliness and warmth rather than the hostility and fear you normally see."